hpobench/container/benchmarks/surrogates/yahpo_gym.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+
+from hpobench.container.client_abstract_benchmark import AbstractBenchmarkClient
+
```

Comment:
Could you please add a second new line here (pep8 Codesytle error) so that the tests finish successfully?

```suggestion


```